app-office/scribus/scribus-1.5.2-r2.ebuild

```diff
@@ -134,6 +135,7 @@ src_configure() {
 		-DWANT_DISTROBUILD=ON
 		-DDOCDIR="/usr/share/doc/${PF}/"
 		-DWANT_GUI_LANG="${langs#;};en"
+		-DWANT_CPP11=$(has_version ">=dev-qt/qtcore-5.7.0" && echo yes || echo no)
```

Comment:
please just do this unconditionally, `has_version` is too much automagic. C++11 should work nonetheless, right, even on older versions?